coretex/cli/modules/node.py

```diff
@@ -287,10 +343,18 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
 
     if verbose:
         config["storagePath"] = clickPrompt("Storage path (press enter to use default)", config_defaults.DEFAULT_STORAGE_PATH, type = str)
-        config["nodeRam"] = clickPrompt("Node RAM memory limit in GB (press enter to use default)", config_defaults.DEFAULT_RAM_MEMORY, type = int)
+
+        cpuCount = clickPrompt("Enter the number of CPUs the container will use (press enter to use default)", config_defaults.DEFAULT_CPU_COUNT, type = int)
+        if cpuCount > cpuLimit:
+            stdEcho(f"WARNING: CPU limit in Docker Desktop ({cpuLimit}) is lower than the configured value ({config['cpuCount']}). Please adjust resource limitations in Docker Desktop settings.")
+            cpuCount = cpuLimit
+        config["cpuCount"] = cpuCount
```

Comment:
This logic is invalid, it should not default to `cpuLimit` in verbose mode. The user must never be able to enter incorrect value using CLI.